client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -207,10 +207,22 @@ function HelpSearchResults( {
 			.filter( ( type, index, arr ) => arr.indexOf( type ) === index );
 
 		return searchResultTypes.map( ( resultType ) => {
+			let displayedResults = results;
+			switch ( resultType ) {
+				case SUPPORT_TYPE_CONTEXTUAL_HELP:
+					displayedResults = results.filter( ( r ) => r.support_type === resultType ).slice( 0, 5 );
+					break;
+				case SUPPORT_TYPE_API_HELP:
+					displayedResults = results.filter( ( r ) => r.support_type === resultType ).slice( 0, 5 );
+					break;
+				case SUPPORT_TYPE_ADMIN_SECTION:
+					displayedResults = results.filter( ( r ) => r.support_type === resultType ).slice( 0, 3 );
```

Comment:
Can you help me elaborate why this has to be `.slice(0, 3)` while the others are `.slice(0, 5)`? I'm asking because we can keep this part untouched if there is no good reason we have to only extract the first 3 results for `SUPPORT_TYPE_ADMIN_SECTION`. If we do, the first two cases can be combined still so it's more concise.

Reply:
This is the UI requirement that existed for the existing component ( shared in the issue description).
Ah yes in mind I just broke it to two cases so that it would be easier for someone to change, but that doesn't seem like much effort come to think of it!